modules/javafx.controls/src/test/java/test/javafx/scene/chart/XYNumberLineChartsTest.java

```diff
@@ -137,6 +143,38 @@ public void testSeriesClearWithoutSymbolsAnimated_8150264() {
         assertEquals(0, XYChartShim.Series_getDataSize(series));
     }
 
+    @Test
+    public void testMinorTicksMatchMajorTicksAfterAnimation() {
+        startAppWithSeries();
+        chart.setAnimated(true);
+        // increase ranges, starting axis animation
+        chart.getData().getFirst().getData().add(new XYChart.Data<>(100, 100));
+        pulse();
+        toolkit.setAnimationTime(1000);
+        NumberAxis yAxis = (NumberAxis)chart.getYAxis();
+
+        List<Double> majorTickYValues = yAxis.getChildrenUnmodifiable().stream()
+                .filter(obj -> obj instanceof Path && obj.getStyleClass().contains("axis-tick-mark"))
+                .flatMap(obj -> ((Path) obj).getElements().stream())
+                .filter(path -> path instanceof MoveTo)
+                .map(moveTo -> ((MoveTo) moveTo).getY())
+                .toList();
+
+        List<Double> minorTickYValues = yAxis.getChildrenUnmodifiable().stream()
+                .filter(obj -> obj instanceof Path && obj.getStyleClass().contains("axis-minor-tick-mark"))
+                .flatMap(obj -> ((Path) obj).getElements().stream())
+                .filter(path -> path instanceof MoveTo)
+                .map(moveTo -> ((MoveTo) moveTo).getY())
+                .toList();
```

Comment:
very clever - I think this is the right approach to obtain the necessary information via public APIs rather than shims.

just need to be aware that only the nodes present in the scene graph can be found this way (I think, please correct me if I am wrong)